hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/type/BasicNestedFieldsTest.java

```diff
@@ -504,17 +452,6 @@ public void test_missingType() {
                 .isInstanceOf(HazelcastSqlException.class);
     }
 
-    @Test
-    public void test_rowComparison() {
-        initDefault();
-
-        assertThatThrownBy(() -> testInstance().getSql().execute("select 1 from test where to_row((this).organization.office)=(3, 'office1')").iterator().next())
```

Comment:
do we stop supporting `ROW` type? or will better support be added in the future?

Reply:
`ROW(v1,v2,...)` operator seems to still return `ROW` type

Reply:
AFAIU, removal of conversion function doesn't remove the type itself :)

Reply:
if so, we should keep second query in this test which does not use `TO_ROW`, maybe also part of other removed tests, if they are supposed to test `ROW` type rather than that `TO_ROW` function